pyorbital/astronomy.py

```diff
@@ -39,55 +40,97 @@
 import datetime
 
 import numpy as np
+from typing import Tuple, Union
 
 from pyorbital import dt2np
 
-F = 1 / 298.257223563  # Earth flattening WGS-84
-A = 6378.137  # WGS84 Equatorial radius
+# Astronomical constants
+AU = 149597870700.0  # Astronomical unit in meters
+EARTH_ flattening = 1 / 298.257223563  # Earth flattening WGS-84
+EARTH_RADIUS = 6378.137  # WGS84 Equatorial radius in km
 MFACTOR = 7.292115E-5
+EARTH_SEMI_MAJOR_AXIS = 1.00000261 * AU  # Earth semi-major axis in meters
+EARTH_ECCENTRICITY = 0.01671123  # Earth orbital eccentricity
+EARTH_YEAR = 365.25636  # Length of Earth year in days
+PERIHELION_DAY = 3  # Day of year with Earth in perihelion (approx.)
 
+# Type alias for time arguments
+TimeLike = Union[dt.datetime, np.datetime64, np.ndarray]
```

Comment:
There is the `numpy.typing` module added in numpy 1.20 I believe. Could this be used to define the dtype of the ndarray? Using something like `npt.ArrayLike[np.datetime64]`?